<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interestingmath</class>
 <widget class="QDialog" name="interestingmath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Интересная математика</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 218, 255);</string>
  </property>
  <widget class="QLabel" name="feedbackLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resource/img/1617299326_20-p-belo-serii-fon-20.png);
font: 20pt &quot;STXihei&quot;;
color: rgb(10, 96, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resource/img/1617299326_20-p-belo-serii-fon-20.png);
font: 18pt &quot;STXihei&quot;;
color: rgb(10, 96, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="optionsLayout"/>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/icons8-назад-50.png</normaloff>:/resource/img/icons8-назад-50.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
